types/net_matches.d.ts

```diff
@@ -1,7 +1,7 @@
 import { NetFields } from './net_fields';
 import { TriggerTypes } from './net_trigger';
 
-export type Matches = { [s in T]?: string };
+export type Matches = { [s: string]: string };
```

Comment:
I am not very sure if this is the right case, but apparently, the type `T` in here is not defined, so cause the error.
@valarnin can you do a review?

Reply:
I believe that this was originally:

```ts
export type Matches<T> = { [s in T]?: string };
```

I can't recall why it's not defined like this now.

Reply:
But then you should change all `Matches` to `Matches<T>` everywhere.
Maybe `export type Matches<T = string> = { [s in T]?: string };` is the intended?

Reply:
Right, it's coming back to me a bit now. `Matches` was originally an extract of field properties but was changed to be the generic any valid object handler. So the change in this PR is fine/accurate here.

Reply:
Actually, slight adjustment. This should instead be as follows, to match the original intention (e.g. properties might not be defined):
```ts
export type Matches = { [key: string]: string | undefined };
```

Reply:
Okay, I'll do that.